### ISSUE TITLE
What is "optimrandomdir_pytorch" in the "MLAS_07_Denoising_Autoencoder.ipynb"

### ISSUE THREAD

**Issue by ccartermices** [GuitarsAI/MLfAS#1]
Hey, I found a variable in the file "MLAS_07_Denoising_Autoencoder.ipynb" called optimrandomdir_pytorch:
`
#optimrandomdir_pytorch.optimizer(model, loss_fn, X, Ypred, iterations=300, startingscale=1.0, endscale=0.0)
Ypred=model(X_train)
#Ypred=Ypred.detach()
print("Ypred=", Ypred)
    
#randdir=True # True for optimization of random direction, False for pytorch optimization
randdir=False
    
if randdir==True:
#optimization of weights using method of random directions:
    optimrandomdir_pytorch.optimizer(model, loss_fn, X_train, Y, iterations=100000, startingscale=0.25, endscale=0.0)
    #--End optimization of random directions------------------------
else:
    for epoch in range(4000):
`
What is it? Please tell me, thx!

**Comment by GuitarsAI** [comment]
Hello,

'optimrandomdir_pytorch' is an optmizer design by Prof. Schuller based on random directions. As it's not covered on this tutorial, it's commented out. 
You can find more information about Prof. Schuller'S random directions optimizer by visiting his github repositories.
